documentation/upgrade/argo-checkout-to-argo-post-purchase.md

```diff
@@ -0,0 +1,13 @@
+# Upgrading from `@shopify/argo-checkout(-react)` to `@shopify/argo-post-purchase(-react)`
+
+In `@shopify/argo-checkout` and `@shopify/argo-checkout-react` version 0.10.x, we have removed the post-purchase extensions temporarily as we work towards broader availability of checkout extensions. The post-purchase APIs you import from `@shopify/argo-checkout` have been moved to `@shopify/argo-post-purchase` (or `@shopify/argo-post-purchase-react` for the React bindings). These new packages start at the same semantic version as the last release of the `@shopify/argo-checkout` package that contained these APIs. To upgrade, first make sure your extension works on the latest version of `@shopify/argo-checkout(-react)`. Once you have confirmed that, swap out your dependency on `@shopify/argo-checkout(-react)` with the matching post-purchase package:
```

Comment:
> These new packages start at the same semantic version as the last release of the `@shopify/argo-checkout` package that contained these APIs.

This is not true anymore because you reset the versions in a more recent commit, right?
But yup, I think we should indeed keep the same versions as it could help people understanding it's really the same code. The only change is the name.

> To upgrade, first make sure your extension works on the latest version of `@shopify/argo-checkout(-react)`.

Should we be more explicit and specify "The latest 0.9.x version"?

Reply:
Good callout about "the latest 0.9.x version", will change that. The post-purchase packages will be the same versions as argo-checkout once I actually publish, I just had to set it to a lower version because our versioning flow asks you to pick a new, higher, version for each new or changed package.